home-assistant-voice.yaml

```diff
@@ -1716,11 +1716,26 @@ voice_assistant:
   on_stt_vad_end:
     - lambda: id(voice_assistant_phase) = ${voice_assist_thinking_phase_id};
     - script.execute: control_leds
+  on_intent_progress:
+    - if:
+        condition:
+          # A nonempty x variable means a streaming TTS url was sent to the media player
```

Comment:
Intent progress contains that streaming: true was passed, not the URL. That one was in run-start